oneanddone/tasks/filters.py

```diff
@@ -40,8 +40,7 @@ class TasksFilterSet(django_filters.FilterSet):
     search = MultiFieldFilter(
         ['name', 'short_description', 'why_this_matters',
          'prerequisites', 'instructions', 'keyword_set__name'],
-        label=_lazy(u'Search for tasks')
-    )
+        label=_lazy(u'Search for tasks'))
```

Comment:
I realize this happened during the back and forth edits, but the parentheses were correct before, so please revert this file to the way it was before this pull request.